Configuration/TypoScript/ContentElement/Element/Accordion.typoscript

```diff
@@ -51,27 +51,30 @@ lib.contentElement.settings.responsiveimages.contentelements {
                 medium = 0.5
             }
             gutters {
-                default = 24
-                xlarge = 24
-                large = 24
-                medium = 24
+                // 50% of gap, which is 1rem=16px
+                default = 8
+                xlarge = 8
+                large = 8
+                medium = 8
             }
             corrections {
-                default = 25
-                xlarge = 25
-                large = 25
-                medium = 25
-                small = 50
-                extrasmall = 50
+                // accordion padding, which is 1.25rem=20px
+                default = 20
+                xlarge = 20
+                large = 20
+                medium = 20
+                small = 40
+                extrasmall = 40
             }
         }
         top {
             corrections {
-                default = 50
-                large = 50
-                medium = 50
-                small = 50
-                extrasmall = 50
+                default = 20
+                xlarge = 20
```

Comment:
@benjaminkott Bugfix, you forgot "xlarge"

Reply:
@SventB can you provide a pull request for this change only? I could rebuild one but then the author will be lost.

Reply:
@benjaminkott That's ok, just fix it :) Creating a new pull request is so much "overhead".

Reply:
Accordion fixes
https://github.com/benjaminkott/bootstrap_package/commit/dcc08bb3c9d07c33aa8ef4635341c974f7512228